packages/cli/src/commands/init.ts

```diff
@@ -1,11 +1,48 @@
+import * as open from 'open';
+import { Answers } from 'inquirer';
 import { prompt } from '../client';
 import { promptIntroArray } from './init.consts';
+import { HttpServer } from '../server';
+import { SERVER_PORT, SERVER_HOST, REDIRECT_ROUTH, API_OAUTH_URL } from '../constants';
+
+let answers: Answers;
 
 export async function initCommand() {
   try {
-    await prompt(promptIntroArray);
+    answers = await prompt(promptIntroArray);
+
+    await gitHubOAuth();
   } catch (error) {
     // eslint-disable-next-line no-console
     console.log(error);
```

Comment:
```suggestion
    console.error(error);
```
This should also remove the need for the eslint-disable here.